cadquery/occ_impl/geom.py

```diff
@@ -849,21 +839,22 @@ def findOutsideBox2D(bb1, bb2):
         return None
 
     @classmethod
-    def _fromTopoDS(cls, shape, tol=None, optimal=False):
+    def _fromTopoDS(cls, shape, tol=None, optimal=True):
```

Comment:
Are you sure you want to default to the most computationally expensive option here? Maybe since it's a bounding box calculation the amount of CPU time consumed is relatively minor?

Reply:
We are using bbox for locating features so I thought using the precise version would be appropriate. I have not seen any runtime issues - if something takes long it is usually related to bool ops or complicated sweeps.